api/schemas/schemas.py

```diff
@@ -13,13 +31,14 @@ class ConnectionBase(BaseModel):
     second_logical_interface_id: int
 
 
-class ConnectionResponse(ConnectionBase):
-    connection_id: int
+class Connection(ConnectionBase):
+    incoming: Settings
+    outgoing: Settings
 
```

Comment:
You need to test but I think if you do the following you don't have to do the `incoming={}`

```
    incoming: Settings = Settings()
    outgoing: Settings = Settings()
```